internal/plugins/ansible/v1/init.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Modifications copyright 2020 The Operator-SDK Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ansible
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+	"sigs.k8s.io/kubebuilder/pkg/plugin/scaffold"
+
+	"github.com/operator-framework/operator-sdk/internal/kubebuilder/cmdutil"
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/v1/scaffolds"
+)
+
+type initPlugin struct {
+	config    *config.Config
+	apiPlugin createAPIPlugin
+
+	doAPIScaffold bool
+
+	// For help text.
+	commandName string
+}
+
+var (
+	_ plugin.Init        = &initPlugin{}
+	_ cmdutil.RunOptions = &initPlugin{}
+)
+
+// TODO(asmacdo) documentation
+func (p *initPlugin) UpdateContext(ctx *plugin.Context) {
```

Comment:
It is missing the examples: https://github.com/operator-framework/operator-sdk/blob/master/internal/plugins/helm/v1/init.go#L51

Reply:
I have a commit that adds this, but I am waiting to push to not block the CI work

Reply:
Since CI has been moving slowly, I'll add this in a followup PR.